phh-securize.sh

```diff
@@ -1,13 +1,22 @@
 #!/system/bin/sh
 
-SYSTEM=/system
-[ -d /sbin/.magisk/mirror/system ] && SYSTEM=/sbin/.magisk/mirror/system
+if [ -e /system/bin/magisk ]
+then
+    # remove bind-mount of phh-su overriding /system/bin/su -> ./magisk
+    umount -l /system/bin/magisk
+    # we need to modify the real system partition
+    MAGISK_MIRROR="$(magisk --path)/.magisk/mirror"
+    SYSTEM=$MAGISK_MIRROR/system
+    MOUNTPOINT_LIST="$MAGISK_MIRROR/system_root $MAGISK_MIRROR/system"
```

Comment:
Please add /system / as a prefix here to ease my mind (i.e. I'm not convincend that magisk does its mirror in 100% of the cases)

Reply:
In these cases, `SYSTEM` would need to be `/system`.

But for `/system/bin/magisk` to exist, `/system/bin` (or `/system`, or `/`) need to be something else than the system partition (tmpfs, overlay, ...).
In all these cases, we want to modify the real system partition : either it's visible as the mirror, or it's hidden and we can't do anything.